executor/executor.cc

```diff
@@ -997,6 +997,8 @@ void write_coverage_signal(cover_t* cov, uint32* signal_count_pos, uint32* cover
 		for (uint32 i = 0; i < cov->size; i++) {
 			cover_data_t pc = cover_data[i] + cov->pc_offset;
 			uint64 sig = pc;
+			if (is_kernel_pc(pc) < 0)
+				exitf("got bad pc: 0x%llx", (uint64)pc);
```

Comment:
This broke gVisor:
2024/06/21 01:16:00 [FATAL] check failed: coverage is not supported: call 0 failed with errno 38

The strace log contains:
I0621 00:56:04.331217  2557365 strace.go:567] [ 672(   2): 672(   2)] syz-executor E write(0x2 pipe:[65], 0x7f8976f570b0 "got bad pc: 0x140000", 0x14)

Reply:
Right. Sent #4916 to fix it.